src/pages/ReportAvatar.tsx

```diff
@@ -20,24 +20,45 @@ type ReportAvatarOnyxProps = {
 
 type ReportAvatarProps = ReportAvatarOnyxProps & StackScreenProps<AuthScreensParamList, typeof SCREENS.REPORT_AVATAR>;
 
-function ReportAvatar({report = {} as Report, policies, isLoadingApp = true}: ReportAvatarProps) {
+function ReportAvatar({report = {} as Report, policies, isLoadingApp = true, route}: ReportAvatarProps) {
     const policy = policies?.[`${ONYXKEYS.COLLECTION.POLICY}${report?.policyID ?? '0'}`];
-    const policyName = ReportUtils.getPolicyName(report, false, policy);
-    const avatarURL = ReportUtils.getWorkspaceAvatar(report);
+    let title;
+    let avatarURL;
+    let fileName;
+    // eslint-disable-next-line rulesdir/no-negated-variables
+    let shouldShowNotFoundPage;
+
+    const shouldUseGroupChatDraft = !!route.params.isNewGroupChat;
+
+    const [groupChatDraft] = useOnyx(ONYXKEYS.NEW_GROUP_CHAT_DRAFT, {initWithStoredValues: shouldUseGroupChatDraft});
+
+    if (shouldUseGroupChatDraft) {
+        avatarURL = groupChatDraft?.avatarUri ?? undefined;
+        fileName = groupChatDraft?.originalFileName ?? undefined;
+        // When user enters custom group name, it typically stored in groupChatDraft.reportName
+        // If that is null then we will use ReportUtils.getGroupChatName to get the name
+        title = groupChatDraft?.reportName ?? ReportUtils.getGroupChatName(groupChatDraft?.participants.map((participant) => participant.accountID) ?? []);
```

Comment:
```suggestion
        title = groupChatDraft?.reportName || ReportUtils.getGroupChatName(groupChatDraft?.participants.map((participant) => participant.accountID) ?? [], true);
```
- Use or operator (`||`) because we want to construct the name if `reportName` is an empty string
-  Pass `true` to `shouldApplyLimit` param for consistency with `ReportUtils.getReportName`

Reply:
Done✅. Added that before also in this [commit](https://github.com/Expensify/App/pull/41586/commits/6d6953f6b6287006d62c838b445daf301453b788) but it was giving eslint error. Added eslint ignore comment.